UnityProject/Assets/Scripts/Core/Chat/EmoteActionManager.cs

```diff
@@ -10,11 +10,11 @@ public class EmoteActionManager : Managers.SingletonManager<EmoteActionManager>
 		[SerializeField] private EmoteListSO emoteList;
 		public EmoteListSO EmoteList => emoteList;
 
-		public static bool HasEmote(string emote, EmoteActionManager instance)
+		public static bool HasEmote(string emote)
```

Comment:
did you test emotes were still working after your changes?

Reply:
yeah, they work fine